Yoast/wordpress-seo
# Introduce the addon manager

## Summary

This PR can be summarized in the following changelog entry:

* N/A - Added the addon manager that works with My-Yoast

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure there are subscriptions active for the site you are testing on. This should be the case if you are testing on `local.wordpress.test`. If not, please contact something from team Platform.
* To test this set all addon to a lower version.
* Go to the updates page from WordPress (Dashboard -> Updates)
* Press the button (Check now? or something like that) at the top
* See there are updates for the Yoast addons available 
* Update a plugin and verify this works.

## Additional pull request
Please also check and merge:
* [x] https://github.com/Yoast/wordpress-seo-premium/pull/2256
* [x] https://github.com/Yoast/wpseo-news/pull/463
* [x] https://github.com/Yoast/wpseo-video/pull/648
* [x] https://github.com/Yoast/wpseo-woocommerce/pull/321
* [x] https://github.com/Yoast/wordpress-seo-local/pull/1404

## UI changes

## Quality assurance


Fixes #12150
